Tests/Source/LinearRegressionModelTester.cpp

```diff
@@ -1,10 +1,25 @@
+#include <type_traits>
 #include <catch.hpp>
 #include "Rover/LinearRegressionModel.hpp"
 #include "Rover/ListTrial.hpp"
 #include "Rover/Sample.hpp"
 
 using namespace Rover;
 
+namespace {
+  template<typename T>
+  struct Conv {
```

Comment:
Avoid the use of abbreviations. Unless it's a universally accepted or very common abbreviation then write out the full name.